### DISCUSSION TITLE
Start a project

### DISCUSSION THREAD

**Discussion by LuginiAndrea** [enable3d/enable3d#133] [Help / Questions] [answered]
Hello, I sawt this framework as the perfect mix of physics and threejs for my project.
I downloaded the file  "enable3d.framework.0.22.0-dev.1.min.js ", and renamed it to lib.js.
I copy-pasted the getting started example and got this error: 
"Uncaught SyntaxError: import not found: Scene3D".

Does anyone know how to fix? If it was a problem with the download or something else.
I'm sorry if it's a dumb question but I searched online and found nothing.

**Comment by yandeu** [comment, marked as answer]
What "getting started example" do you mean?

Have a look at this example:
https://enable3d.io/examples/standalone-mode.html

You also need the 3 ammo.js files. Either from kripken or MozillaReality:
https://github.com/enable3d/enable3d-website/tree/master/src/lib/ammo

**Reply by LuginiAndrea** [reply]
I mean this example: https://enable3d.io/docs.html#basic-setup 
When I get the 3 files where should I put them? I only installed the file from the bundle, not the entire repo

**Reply by yandeu** [reply]
I just created a very simple starter template 👍 
https://github.com/enable3d/simple-starter

**Reply by LuginiAndrea** [reply]
That worked, great. 
Thank you

**Reply by Ryan-McKee2001** [reply]
> I just created a very simple starter template 👍
> https://github.com/enable3d/simple-starter

Hi there!, I tried using this myself as I am currently creating a project and would like to use enable3d but when I tried to open the index I had an "enable 3d is not defined" console error. Thanks in advance.

**Reply by yandeu** [reply]
I have just updated the instructions to run the example.
See: https://github.com/enable3d/simple-starter#readme

**Reply by Ryan-McKee2001** [reply]
thank you.
